ui.apps/src/main/javascript/minicart/src/components/CreateAccount/__test__/createAccount.test.js

```diff
@@ -40,7 +40,7 @@ describe('<CreateAccount>', () => {
         expect(asFragment()).toMatchSnapshot();
     });
 
-    it('submits the form properly', async () => {
+    it.skip('submits the form properly', async () => {
```

Comment:
Is this intentional here?

Reply:
Yes, it is. The test is flaky and we have a JIRA issue to investigate it.